feature_flag.md

```diff
@@ -4,38 +4,55 @@
 **[this](https://martinfowler.com/articles/feature-toggles.html)**
 **helpful article on the topic.**
 
-## Best practises
+At Companies House we use feature flagging to enable us to deliver features
```

Comment:
Might also be worth mentioning that feature flags facilitate 'trunk-based' development - short-lived branches with fewer code changes (easier to review) that can be merged into 'main'.